scripts/zones/Davoi/npcs/Jar.lua

```diff
@@ -14,18 +14,19 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-    local powerSandalsID = 13012
+    if
+        player:getVar('Quest[4][25]Prog') == 0 or
+        player:hasItem(xi.items.POWER_SANDALS)
+    then
+        player:messageSpecial(ID.text.NOTHING_OUT_OF_ORDINARY)
+        return
+    end
 
-    -- Give Player Power Sandals if they don't have them
-    if not player:hasItem(powerSandalsID) then
-        if player:getFreeSlotsCount() == 0 then
-            player:messageSpecial(ID.text.ITEM_CANNOT_BE_OBTAINED, powerSandalsID)
-        else
-            player:addItem(powerSandalsID)
-            player:messageSpecial(ID.text.ITEM_OBTAINED, powerSandalsID)
-        end
+    if player:getFreeSlotsCount() == 0 then
+        player:messageSpecial(ID.text.ITEM_CANNOT_BE_OBTAINED, xi.items.POWER_SANDALS)
     else
-        player:messageSpecial(ID.text.NOTHING_OUT_OF_ORDINARY)
+        player:addItem(xi.items.POWER_SANDALS)
+        player:messageSpecial(ID.text.ITEM_OBTAINED, xi.items.POWER_SANDALS)
```

Comment:
No, from what I've seen the Jar has always been static; I didn't attempt to update its position like the quest suggests, but I could look into it if it felt worthwhile

Reply:
Added the ability for it to move randomly or after someone clicks it; likewise I made it so that it spawns in one of the several locations listed on FFXIclopedia